bin/elixir

```diff
@@ -69,8 +69,16 @@ ERL=""
 
 # Stores erl arguments preserving spaces/quotes (mimics an array)
 erl () {
-  eval "E$E=\$1"
-  E=$(($E + 1))
+  eval "E${E}=\$1"
+  E=$((E + 1))
+}
+
+# Checks if a line starts with char starts_with "$LINE", "$CHAR"
+starts_with () {
+  case $1 in
+    "$2"*) true;;
+    *) false;;
+  esac
```

Comment:
I indented this and one-lined the comment.

Reply:
ok! great, and thank you @yordis for your input